app/components/navbar.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class NavbarComponent extends Component {
+  @tracked isNavOpen = false;
+
+  HOME_PAGE_URL = 'http://realdevsquad.com/';
+
+  siteURLs = [
```

Comment:
Let's call it `RDS_APPS_MAPPING`? (special focus on uppercase since it's a constant)

Reply:
ok will change it. This name is great and well explains its purpose